test/multiverse/suites/httpx/httpx_instrumentation_test.rb

```diff
@@ -0,0 +1,114 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'httpx'
+require 'newrelic_rpm'
+require 'http_client_test_cases'
+require 'uri'
+require_relative '../../../../lib/new_relic/agent/http_clients/httpx_wrappers'
+require_relative '../../../test_helper'
+
+class PhonySession
+  include NewRelic::Agent::Instrumentation::HTTPX
+
+  def initialize(responses = {})
+    @responses = responses
+  end
+end
+
+class HTTPXInstrumentationTest < Minitest::Test
+  include HttpClientTestCases
+
+  # TODO: make sure our transaction level and segment level error
+  #       handling for HTTPX is working as desired
```

Comment:
Do you want to resolve this TODO before merging?

Reply:
Error handling and testing updated with [ee243b9](https://github.com/newrelic/newrelic-ruby-agent/pull/2278/commits/ee243b97be5fe34ebd43b977588603cbf2b027e2), no more TODOs remain. Thank you!